### PR TITLE
feat: Sign In Page Design Update size/S

### PR DESCRIPTION
**Pull request by jayfyle**
# Design Update for sign in page:

sign_in_3

### CODE REVIEW

File: src/app/auth/sign-in/sign-in.page.scss [hunk 1]
```diff
@@ -1,8 +1,14 @@
+@import '../../../theme/colors.scss';
 $sign-in-header: #220033;
 $form-header: #000;
 $form-subheader: #4a4a4a;
 $password-icon: #b9beba;
-$secondary-cta-border: #e0e0e0;
+$secondary-cta-border: #a9acbc;
```
Comment (jayfyle) [inline src/app/auth/sign-in/sign-in.page.scss:6]: Changed colour as it was in the figma file

File: src/app/auth/sign-in/sign-in.page.scss [hunk 2]
```diff
@@ -29,7 +35,10 @@ $secondary-cta-border: #e0e0e0;
   }
 
   &--form {
-    padding: 24px;
+    padding: 40px 24px;
+    margin: 16px;
+    background-color: $white;
+    border-radius: 8px;
```
Comment (jayfyle) [inline src/app/auth/sign-in/sign-in.page.scss:41]: Border-radius in the file was 8px and padding was 40 and 24

File: src/app/auth/sign-in/sign-in.page.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<ion-content>
+<ion-content class="background">
```
Comment (Dimple16) [inline src/app/auth/sign-in/sign-in.page.html:1]: follow the bem, add a class inside `.sign-in` something like `--page-container`

Comment (Dimple16) [reply src/app/auth/sign-in/sign-in.page.html:1]: and use it here as `class="sign-in--page-container"`

File: src/app/auth/sign-in/sign-in.page.scss (hunk 1 above)
Comment (Dimple16) [reply src/app/auth/sign-in/sign-in.page.scss:6]: use the colors from the `colors.scss` file, refer the `/themes` folder

Comment (Dimple16) [reply src/app/auth/sign-in/sign-in.page.scss:6]: `$grey-light` is already defined, you can use the same,

File: src/app/auth/sign-in/sign-in.page.scss [hunk 4]
```diff
@@ -1,8 +1,14 @@
+@import '../../../theme/colors.scss';
 $sign-in-header: #220033;
 $form-header: #000;
 $form-subheader: #4a4a4a;
 $password-icon: #b9beba;
-$secondary-cta-border: #e0e0e0;
+$secondary-cta-border: #a9acbc;
+
+.background {
```
Comment (Dimple16) [inline src/app/auth/sign-in/sign-in.page.scss:8]: this will go inside the `.sign-in` block

File: src/app/auth/sign-in/sign-in.page.scss [hunk 5]
```diff
@@ -29,7 +35,10 @@ $secondary-cta-border: #e0e0e0;
   }
 
   &--form {
-    padding: 24px;
+    padding: 40px 24px;
```
Comment (Dimple16) [inline src/app/auth/sign-in/sign-in.page.scss:38]: why `40px`?  is it 40 in design?

File: src/app/auth/sign-in/sign-in.page.scss [hunk 6]
```diff
@@ -71,8 +80,20 @@ $secondary-cta-border: #e0e0e0;
...
       letter-spacing: 1.6px;
       border: 1px solid $secondary-cta-border;
+      background-color: $white;
     }
     margin-top: 24px;
+    &-btn {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      gap: 10px;
```
Comment (Dimple16) [inline src/app/auth/sign-in/sign-in.page.scss:90]: check if this is responsive and looks fine in all the resolutions of devices

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

**Review by Dimple16 — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**